influent-java/src/main/java/influent/forward/NioForwardConnection.java

```diff
@@ -93,15 +116,38 @@
                        final long chunkSizeLimit,
                        final int sendBufferSize,
                        final boolean keepAliveEnabled,
```

Comment:
This means TCP keepalive configuration……

Reply:
Now, I cannot decide how to configure TCP keepalive and handshake's keepalive.
(I'm thinking that it is OK that TCP keepalive is always enabled and its configuration will be removed from Influent).

How about setting `keepalive` on HELO message `true` and adding TODO comment to `generateHelo` method?

Reply:
OK, I will add TODO comment and set `keepalive` on HELO message `true`.